runtime/CMakeLists.txt

```diff
@@ -114,6 +114,10 @@ if(NOT APPLE)
   install_helper(
     FILES ${CMAKE_CURRENT_SOURCE_DIR}/nvim.png
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/128x128/apps)
+
+  install_helper(
+    FILES ${CMAKE_CURRENT_SOURCE_DIR}/neovim.ico
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime)
```

Comment:
I wonder why this and the "nvim.png" case above, are in (NOT APPLE) block. Doesn't hurt to install them on all systems, and avoids some unnecessary differences.